deployment-templates/compute-engine/forseti-instance.py.schema

```diff
@@ -80,3 +80,15 @@ properties:
     type: integer
     default: 3306
     description: The port for the database.
+
+  network-host-project:
+    type: string
+    description: the id of the shared network host project
```

Comment:
For clarity, can you please rephrase this description:
```The id of the project that is hosting the shared network resources.```

Reply:
Complete